addons/portal/wizard/portal_share.py

```diff
@@ -86,8 +87,9 @@ def _send_signup_link(self, partners=None):
             self.resource_ref.message_post_with_source(
                 'portal.portal_share_template',
                 render_values={'partner': partner, 'note': self.note, 'record': self.resource_ref,
-                        'share_link': share_link},
-                subject=_("You are invited to access %s", self.resource_ref.display_name),
+                        'share_link': share_link,
+                        'record_description': self.env['ir.model']._get(self.resource_ref._name).name.lower()},
```

Comment:
Note that in mail and sub-addons we call it generally ``model_description`` to know it is the the current record's model description. Not a standard, just proposing.

Reply:
Okay, we will do the same then. 😁